tutorials/Land/Soil/Water/equilibrium_test.jl

```diff
@@ -189,7 +189,7 @@ m = LandModel(
 # # Specify the numerical configuration and output data.
```

Comment:
was the solution you derived valid for over-saturated btw or only for saturated? I think yes, because we used psi = theta - porosity / S , but just wanted to make sure

Reply:
thank you!! The original version of this was just for saturated, but this updated one now applies for both

Reply:
cool. just to clarify, both meaning unsaturated and saturated? i guess i meant more if the soln could handle oversaturated?